docs/developer-docs/latest/setup-deployment-guides/deployment/hosting-guides/digitalocean-app-platform.md

```diff
@@ -4,122 +4,320 @@ description: Learn in this guide how to deploy your Strapi application on Digita
 
-Click on the Add a Database button. You can create a development PostgreSQL database while testing your application. Alternatively, you can add a Managed Database that you have created.
+1. Add the database, URL, and NODE_ENV variables to the global environment table:
 
-Name your database (default name is `db`). Whatever you name your database here is what you should use in the environment variables in Step 5 above. For instance, we name the database `db` and we use the environment variable value: `${db.DATABASE_HOST}`
+    | Variable name     | Value          |
+    |-------------------|----------------|
+    | URL               | ${APP_URL}     |
+    | DATABASE_HOST     | ${db.HOSTNAME} |
+    | DATABASE_PORT     | ${db.PORT}     |
+    | DATABASE_NAME     | ${db.DATABASE} |
+    | DATABASE_USERNAME | ${db.USERNAME} |
+    | DATABASE_PASSWORD | ${db.PASSWORD} |
+    | NODE_ENV          | production     |
 
-Click "Next".
-### Step 7. Add Strapi Upload Provider for Digital Ocean Spaces
+2. Click **Save**.
+3. Navigate to the *Settings* menu and select the Strapi application component.
+4. Scroll to the *Environment Variables* and click **edit**.
+5. Add the key-value pairs to the component environment variables table:
 
-```bash
-yarn add strapi-provider-upload-do
+    | Variable name    | value                               |
+    |------------------|-------------------------------------|
+    | APP_KEYS         | "unique user-generated secrets here"|
+    | API_TOKEN_SALT   | "unique user-generated secret here" |
+    | ADMIN_JWT_SECRET | "unique user-generated secret here" |
+    | JWT_SECRET       | "unique user-generated secret here" |
+
+6. Click **Save**.
+
+::: note
+It is possible to copy the secrets from the local `.env` file or to generate new secrets using a random secret generator. Examples can be found at [Openssl](https://www.openssl.org/).
+:::
+
+## Deploy and access a Strapi application
+
+When the preceding steps are completed DigitalOcean should automatically try to build and deploy the application. The Strapi admin panel is accessed at `{your App domain}/admin` once the application is successfully deployed.
+
+::: tip
+If the build does not start, click the **Actions** button in the upper right and select *force rebuild and deploy*.
+:::
+
+## Add a managed database
+
+DigitalOcean managed databases are a production-scale database solution for a deployed Strapi application. Switching from a development database to a managed database requires modifying the Strapi application and modifying the settings on DigitalOcean:
+
+- removing the development database and database environment variables,
```

Comment:
I see 4 bullet points here while we have 2 sub-sections. Maybe it would be best to rephrase a bit, to easily understand that modifying the Strapi application consists in removing env. vars, installing the dependency, and modifying the production file.

Reply:
Agreed. Especially since in the last section ("Modify settings on DigitalOcean") we have no introduction or explanation, only a procedure, so it makes it even unclear. I would also rework this list + provide more explanations in that last section, just to make sure people know what's happening.

Reply:
I have reworked this into 2 separate sections with more detail.